alpha/lib/model/UserEntryPeer.php

```diff
@@ -84,8 +81,12 @@ public static function getEntryIdsByFilter($limit, $offset = 0, $filter = null,
 		$userEntryCriteria->setLimit($limit);
 		$userEntryCriteria->setOffset($offset);
 		
-		$stmt = self::doSelectStmt($userEntryCriteria);
-		$ids = $stmt->fetchAll(PDO::FETCH_COLUMN);
+		$result = UserEntryPeer::doSelect($userEntryCriteria);
+		
+		$ids = array();
+		foreach ($result as $res) {
```

Comment:
We probably want to put the curly brackets at the next line

Other then that looks OK